pkg/cli/admin/update/update.go

```diff
@@ -52,64 +52,77 @@ func NewOptions(streams genericclioptions.IOStreams) *Options {
+			known to be supported as updates from the current version.
 
-			Passing --to=VERSION or --to-image=IMAGE will upgrade the cluster to one of the available
-			updates or report an error if no such version exists. The cluster will then upgrade
+			Passing --to=VERSION or --to-image=IMAGE will update the cluster to one of the available
+			updates or report an error if no such version exists. The cluster will then update
 			itself and report status that is available via "oc get clusterversion" and "oc describe
 			clusterversion".
 
-			Passing --to-multi-arch will upgrade the cluster from a single-architecture to a
+			Passing --to-multi-arch will update the cluster from a single-architecture to a
 			multi-architecture cluster at the current version.
 
 			If there are no versions available, or a bug in the cluster version operator prevents
 			updates from being retrieved, --to-image may be combined with the more powerful and
-			dangerous --allow-explicit-upgrade. This instructs the cluster to upgrade to the contents
-			of the specified release image, regardless of whether that upgrade is known to be
+			dangerous --allow-explicit-update. This instructs the cluster to update to the contents
+			of the specified release image, regardless of whether that update is known to be
 			recommended for the current version. While rolling back to a previous patch (z stream) version
-			(4.1.2 -> 4.1.1) may be safe, upgrading more than one minor version ahead (4.1 -> 4.3) or
+			(4.1.2 -> 4.1.1) may be safe, updating more than one minor version ahead (4.1 -> 4.3) or
 			downgrading one minor version (4.2 -> 4.1) is likely to cause data corruption or to
 			completely break a cluster.
 
-			There are two layers of upgrade guards: client-side and cluster-side.
+			There are two layers of update guards: client-side and cluster-side.
 
-			Client-side guards include checks for whether the cluster is already being upgraded, or if
+			Client-side guards include checks for whether the cluster is already being updated, or if
 			the cluster is reporting a failure.  It is usually best to give these conditions time to
 			resolve, or to actively work to resolve them.  But if you decide to trigger the update
-			regardless of these concerns, use --allow-upgrade-with-warnings.
+			regardless of these concerns, use --allow-update-with-warnings.
 
-			Cluster-side guards include checks for release verification and upgradeable conditions.
+			Cluster-side guards include checks for release verification and updateable conditions.
 			Again, it is usually best to give these conditions time to resolve, or to actively work to
 			resolve them.  But if you decide to trigger the update regardless of these concerns,
 			use --force, which is passed through to ClusterVersion's spec.desiredUpdate.force.
 		`),
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			if cmd.CalledAs() != "update" {
```

Comment:
```
❯ oc adm update channel 
oc adm update channel is deprecated.  Use 'oc adm update channel ...' instead
info: Cluster channel is already clear (no change)
```
hmm.. we're not deprecating the channel subcommand right?

Reply:
Thanks!  Fixed in 003a50de3 -> 5f78beb41:

```console
$ ./oc adm upgrade channel
oc adm upgrade channel is deprecated.  Use 'oc adm update channel ...' instead
error: You are requesting to clear the update channel. The current channel "candidate-4.13" is one of the available channels, you must pass --allow-explicit-channel to continue
```

Reply:
retested - `update channel` still triggering a false positive!
```
❯ oc adm update channel
oc adm upgrade channel is deprecated.  Use 'oc adm update channel ...' instead
info: Cluster channel is already clear (no change)
```

Reply:
btw, there's a double space in the middle - intentional?